src/sentry/static/sentry/app/components/group/releaseStats.tsx

```diff
@@ -13,15 +13,19 @@ type Props = {
   environments: Environment[];
   allEnvironments: Group | undefined;
   group: Group | undefined;
+  currentRelease: any | null | undefined; // TODO(ts)
 };
 
-const GroupReleaseStats = ({
-  group,
-  organization,
-  project,
-  environments,
-  allEnvironments,
-}: Props) => {
+const GroupReleaseStats = (props: Props) => {
+  const {
+    organization,
+    project,
+    environments,
+    allEnvironments,
+    group,
+    currentRelease,
+  } = props;
+
```

Comment:
Is there a reason for changing this?

Reply:
Was tweaking the entire component around but I undid that. No reason to change it now so reverting back.